### ISSUE TITLE
new feature require:add new job from remote

### ISSUE THREAD

**Issue by rqx110** [guryanovev/CrystalQuartz#14]
want to add new job from web, and the server can schedule it to execute.
Also it can modify trigger properties(eg. interval time)

**Comment by ChesterZengJian** [comment]
Yes, i want to add a new job for difference  JobDataMap，and then it can modify trigger properties。How to implement it?
